core/src/main/java/com/googlecode/psiprobe/tools/url/UrlParser.java

```diff
@@ -34,6 +34,13 @@
 
   /**
    * Instantiates a new url parser.
+   */
+  public UrlParser() {
```

Comment:
You are right.  I should have avoided changing source code when adding these tests in general at this level.  There were a number I couldn't get working anyways so I added ```@ignore``` on them but I didn't in this case based on poor knowledge.  I should have done that with these constructor changes as well.  It certainly would have been easier to bring this in with zero source changes so review wasn't necessary then make adjustments later for additional support.  I don't know that i want to hit the revert button but I certainly will go back through all these in next couple of days and revert the specific source code changes in question.  I'll do that prior to pulling in the packaging changes.

Thanks for reviewing all this stuff.  Other than questionable changes, so far so good.  I'll get a little more deeper testing over next couple of days as well.  Do you know if there are use cases out there presently where people are extenting this library other than drop and go?